evap/evaluation/views.py

```diff
@@ -159,7 +159,6 @@ def legal_notice(request):
 
 
 @require_POST
-@login_required
```

Comment:
For a nice git history, we probably want to have one, well documented commit that removes the four usages (you have that) as well as the implementation (think it's still in the code), and links the PR where it should have been removed in the past.